tests/test_configurations/test_manager.py

```diff
@@ -72,12 +71,23 @@
   ethereum: ethereum_private_key.txt
 default_routing: {}
 """
-)
+
+DUMMY_SKILL_OVERRIDE = """
+public_id: dummy_author/test_skill:0.1.0
+type: skill
+models:
+  scaffold:
+    args:
+      recursive:
+        hello: world
+"""
```

Comment:
what happens when there are duplicate keys?

Reply:
The loader with raise an error